Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserThumbnailView.cpp

```diff
@@ -71,6 +76,82 @@ namespace AzToolsFramework
                     emit showInFolderTriggered(indexData);
                 });
 
+            connect(
+                m_thumbnailViewWidget,
+                &AzQtComponents::AssetFolderThumbnailView::contextMenu,
+                this,
+                [this](const QModelIndex& index)
+                {
+                    if (index.isValid())
+                    {
+                        QMenu menu(this);
+                        const AssetBrowserEntry* entry = index.data(AssetBrowserModel::Roles::EntryRole).value<const AssetBrowserEntry*>();
+                        AZStd::vector<const AssetBrowserEntry*> entries{ entry };
+                        AssetBrowserInteractionNotificationBus::Broadcast(
+                            &AssetBrowserInteractionNotificationBus::Events::AddContextMenuActions, this, &menu, entries);
+
+                        if (!menu.isEmpty())
+                        {
+                            menu.exec(QCursor::pos());
+                        }
+                    }
+                    else if (!index.isValid() && m_assetTreeView)
+                    {
+                        m_assetTreeView->OnContextMenu(QCursor::pos());
+                    }
+                });
+
+            connect(m_thumbnailViewWidget, &AzQtComponents::AssetFolderThumbnailView::afterRename, this, &AssetBrowserThumbnailView::AfterRename);
+
+              if (AzToolsFramework::IsNewActionManagerEnabled())
+              {
+                  if (auto hotKeyManagerInterface = AZ::Interface<AzToolsFramework::HotKeyManagerInterface>::Get())
+                  {
+                      // Assign this widget to the Editor Asset Browser Action Context.
+                      hotKeyManagerInterface->AssignWidgetToActionContext(
+                          EditorIdentifiers::EditorAssetBrowserActionContextIdentifier, this);
+                  }
+              }
```

Comment:
This is already being done in the tree view, so we shouldn't register again in the thumbnail view. Ideally, this would be refactored to move the shortcuts all to the top level asset browser widget instead of the individual views and be registered with the new action manager, but that can be done in a future pass.